website/docs/cloud-docs/workspaces/health.mdx

```diff
@@ -60,6 +61,19 @@ Terraform Cloud pauses health assessments if the latest run ended in an errored
 
 Health assessments may take longer to complete when you enable health assessments in many workspaces at once or your workspace contains a complex configuration with many resources.
 
+### On Demand Assessments
+
+-> **Note:** On Demand Assessments are only available through browser sessions. If the workspace requirements are not satisfied, you should see one or more error messages on the page indicating the blockers to enable health assessments. The button will be disabled until the requirements are satisfied.
+
+
+If you are an administrator for the workspace and all [workspace requirements](/terraform/cloud-docs/workspaces/health#workspace-requirements) are satisfied, a **Start health assessment** button is available on the Health pages with Ready status.
+
+  ![On Demand Assessments Screenshot: Start health assessment button](/img/docs/on-demand-assessments.png)
```

Comment:
looking at the UI, i think we can remove this screenshot since the button is prominent enough.